composer/checkpoint/state_dict.py

```diff
@@ -88,7 +88,7 @@ def get_model_state_dict(
             log.debug('Calling model.state_dict() for non-FSDP model...')
             model_state_dict = model.state_dict()
         if isinstance(model, DistributedDataParallel):
-            nn.modules.utils.consume_prefix_in_state_dict_if_present(model_state_dict, 'module.')
+            nn.modules.utils.consume_prefix_in_state_dict_if_present(model_state_dict, 'module.')  # type: ignore
```

Comment:
why is this needed? are you sure this isnt API change?

Reply:
It is an API change (they've been adding `__all__` to their files) but they didn't expose their utils correctly